modules/terraform/show_test.go

```diff
@@ -0,0 +1,153 @@
+package terraform
+
+import (
+	"io/ioutil"
+	"net"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/files"
+	"github.com/gruntwork-io/terratest/modules/terraform/jsonplan"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Thanks for adding unit tests! However, I have a nit: can you also provide instructions on how the fixtures were generated so that in the future when we need to update them to the newer plan file format versions, it will be easy to do?